internal/service/environment.go

```diff
@@ -23,3 +40,36 @@ func (e *Environment) ListEnvironments(ctx *context.Context, request *environmen
 
 	return response, nil
 }
+
+// CreateEnvironment creates environment
+func (e *Environment) CreateEnvironment(ctx *context.Context, request *environment.CreateEnvironmentRequest) error {
+	conn, requestCtx, err := grpcClient(ctx)
+	if err != nil {
+		return err
+	}
+	client := environment.NewEnvironmentServiceClient(conn)
+	stream, err := client.CreateEnvironment(*requestCtx, request)
+	if err != nil {
+		return err
+	}
+
+	log.Info("Creating environment...")
+	spinner := spinner.New(spinner.CharSets[SpinnerType], SpinnerDelay)
+	spinner.Color(SpinnerColor, SpinnerStyle)
+
+	for {
+		response, err := stream.Recv()
+		spinner.Stop()
+		if err != nil {
+			if err == context.Canceled || err == io.EOF {
+				break
+			}
+			return err
+		}
+		spinner.Prefix = fmt.Sprintf(" %s  ", response.Message)
+		spinner.Start()
+	}
+	log.Info("Environment created successfully")
```

Comment:
The stream will always exit successfully so this log will be always printed. We need to check the status to know whether the environment was created or not?

Reply:
Fixed
PS: Stream fails with error whenever deploy throws creation failed error.